src/sagemaker/workflow/pipeline.py

```diff
@@ -263,7 +266,7 @@ def format_start_parameters(parameters: Dict[str, Any]) -> List[Dict[str, Any]]:
     return [{"Name": name, "Value": str(value)} for name, value in parameters.items()]
 
 
-def interpolate(request_obj: RequestType) -> RequestType:
+def interpolate(request_obj: RequestType, **kwargs) -> RequestType:
```

Comment:
is there a reason to use kwargs here? Seems like you are only adding one parameter.

Reply:
It was my way of making the arg optional since it won't be included in every `interpolate` call, but I can get rid of the kwargs